docuware-connector-product/README.md

```diff
@@ -44,7 +44,7 @@ Before any interactions between the Axon Ivy Engine and DocuWare services can be
 ```
 Variables:
   
-  docuware-connector:
+  docuwareConnector:
   
     host: <myhost>.docuware.cloud
 
```

Comment:
the 'fileCabinetId' is missing here in the readme.
to keep it sync automatically, please use the latest template 'pom.xml' for product and inject the variable listing via @variables.yaml@ as in the newer connectors. 
you may refer to https://github.com/axonivy-market/openai-connector/blob/master/openai-connector-product/README.md?plain=1#L32 for an example

Reply:
Wow, It's a new thing to me. I would handle it right away